sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -210,55 +236,110 @@ class AnalyzeHealthcareEntitiesResultItem(DictMixin):
     def __init__(self, **kwargs):
         self.id = kwargs.get("id", None)
         self.entities = kwargs.get("entities", None)
+        self.relations = kwargs.get("relations", None)
         self.warnings = kwargs.get("warnings", [])
         self.statistics = kwargs.get("statistics", None)
         self.is_error = False
 
-    @staticmethod
-    def _update_related_entities(entities, relations_result):
-        relation_dict = {}
-        for r in relations_result:
-            _, source_idx = _get_indices(r.source)
-            _, target_idx = _get_indices(r.target)
-
-            if entities[source_idx] not in relation_dict.keys():
-                relation_dict[entities[source_idx]] = {}
-
-            if entities[target_idx] not in relation_dict.keys():
-                relation_dict[entities[target_idx]] = {}
-
-            if r.bidirectional:
-                relation_dict[entities[target_idx]][entities[source_idx]] = r.relation_type
-
-            relation_dict[entities[source_idx]][entities[target_idx]] = r.relation_type
-
-        for entity in entities:
-            if entity in relation_dict.keys():
-                entity.related_entities.update(relation_dict[entity])
 
     @classmethod
     def _from_generated(cls, healthcare_result):
         entities = [HealthcareEntity._from_generated(e) for e in healthcare_result.entities] # pylint: disable=protected-access
-        if healthcare_result.relations:
-            cls._update_related_entities(entities, healthcare_result.relations)
+        relations = [HealthcareRelation._from_generated(r, entities) for r in healthcare_result.relations]  # pylint: disable=protected-access
 
         return cls(
             id=healthcare_result.id,
             entities=entities,
+            relations=relations,
             warnings=healthcare_result.warnings,
-            statistics=healthcare_result.statistics
+            statistics=healthcare_result.statistics,
```

Comment:
is the `,` needed? idk python :P

Reply:
it doesn't really matter, it's pretty common to have a trailing comma, since adding something else to the list becomes easier later on